proto/lfb/auth/v1beta1/auth.proto

```diff
@@ -1,6 +1,9 @@
 syntax = "proto3";
 package lfb.auth.v1beta1;
 
+import "lfb/crypto/ed25519/keys.proto";
+import "lfb/crypto/multisig/keys.proto";
+import "lfb/crypto/secp256k1/keys.proto";
 import "cosmos_proto/cosmos.proto";
 import "gogoproto/gogo.proto";
 import "google/protobuf/any.proto";
```

Comment:
"google/protobuf/any.proto" seems to be unused, so is would be ok to delete it.

Reply:
Applied